.github/workflows/sandbox.yml

```diff
@@ -2,31 +2,46 @@ name: Build & Push Sandbox Docker Image
 
 on:
   pull_request:
-  create:
-    tags:
-      - v*
+  release:
```

Comment:
@EngHabu: How do you feel about using this trigger as opposed to `create`? This seems better if y'all are going to be cutting releases.

Reply:
That's fine... is there a reason for the change? will one make it better for forking than the other?

Reply:
I think they're both ok for forking. But this one better conveys the intent imo, since we explicitly cut releases for the repo. I'm fine with either.

Reply:
awesome... release creation is fine!